docs/GettingStarted/install.md

```diff
@@ -57,128 +57,213 @@ cd ${HOME}/opt/self/test
 ctest
 ```
 
+### Once v0.0.1 is released 
+The easiest way to get started is to use the [spack package manager](https://spack.io). The spack package manager provides you with an easy command line interface to install research software from source code with all of its dependencies. 
 
+!!! note
```

Comment:
```suggestion
**! NOTE:**
```

Reply:
Sticking to the conventions in the [mkdocs admonitions documentation] (https://squidfunk.github.io/mkdocs-material/reference/admonitions/)